CHANGELOG.rst

```diff
@@ -13,6 +13,7 @@ Change Log
 
 Unreleased
 ~~~~~~~~~~
+* Add Maple support.
```

Comment:
@BbrSofiane I may be wrong, but this change removes Python 2 and Django 2.x compatibility too.

Reply:
Yes, you're right